power-control-x86/src/power_control.cpp

```diff
@@ -1805,6 +1805,9 @@ int main(int argc, char* argv[])
         return -1;
     }
 
+    // Check if we need to start the Power Restore policy
+    power_control::powerRestorePolicyCheck();
```

Comment:
This should be moved after if (!power_control::setGPIOOutput("ASSERT_BMC_READY", 1, gpioLine)).

I know I told you to move it before the BMC_READY statement during our call.
But after looking through further, power restore policy should be executed after BMC_READY.
Apologies for not telling you earlier.